tests/test_basic.c

```diff
@@ -0,0 +1,237 @@
+/* vim:set ft=c ts=2 sw=2 sts=2 et cindent: */
+/*
+ * Copyright 2014 Michael Steinert
```

Comment:
Since you're the author, please update with the current year (2017), and your name.

Reply:
Done